curricula/templates/curricula/partials/lesson.html

```diff
@@ -26,7 +27,7 @@ <h1>Teaching Guide</h1>
                 {% for activity in lesson.activity_set.all %}
                     <div class="activity">
                         <h2>{{ activity.name }}{% if activity.time %} ({{ activity.time }}){% endif %}</h2>
-                        {{ activity.content|richtext_filters|safe }}
+                        {{ activity.content|with_unit:unit|richtext_filters|safe }}
```

Comment:
after updating http://localhost:8000/csf-18/coursec/4/ to contain this placeholder, what url could I navigate to in order to exercise this line of code? or should I omit this for now?

Reply:
Not sure I understand the question. Am I correct in reading that I'd use this tag like `[link to script](//studio.code.org/s/{{{unit_name}}})`

Reply:
Correct, that's how you'd use this new syntax within an activity's content.

The background for my question is, when I use the admin interface to edit an activity's content to contain the `{{{unit_name}}}` placeholder, and then I view it at a url like http://localhost:8000/csf-18/coursec/4/ , I can see that the substitution is working, and I can tell by debugging that the code which is doing the substitution is at   https://github.com/mrjoshida/curriculumbuilder/pull/58/files#diff-a1fb870305086ceb34101808f693a4c2R195

I also made this code change on partials/lesson.html because it appears to render activity.content, but I can't figure out where (if anywhere) the activity content rendered by this partial would show up within my local curriculum builder. If it shows up somewhere, I'd like to verify that the substitution is working.

From looking at the code, it seems like this only ever shows up in unit_view or unit_compiled, which in turn only get used to generate URLs. But I'm new to django, so I just wanted some confirmation that I'm not missing something.

Reply:
a math teacher might ask me to show my work 😛 
* partials/lesson.html appears to only be rendered here:
https://github.com/mrjoshida/curriculumbuilder/blob/a1d7f13701c6eeea648b12f1a88edbd8c74986e4/curricula/templates/curricula/unit_lessons.html#L209-L214
* unit_lessons.html appears to only be rendered in https://github.com/mrjoshida/curriculumbuilder/blob/9736e85db46ded446f5b8c948ac73286cf6daaac/curricula/views.py#L115 and https://github.com/mrjoshida/curriculumbuilder/blob/9736e85db46ded446f5b8c948ac73286cf6daaac/curricula/views.py#L404
* `unit_view` and `unit_compiled` appear to only be mentioned as inputs to things that generate urls.

Reply:
the the lessons partial is used in the unit_compiled view is used to generate the unit PDFs through eg  http://localhost:8000/csf-18/coursec/compiled/

Reply:
Perfect, thank you Josh! That is working:
<img width="809" alt="screen shot 2018-12-19 at 1 31 50 pm" src="https://user-images.githubusercontent.com/8001765/50249324-8308a080-0392-11e9-874e-dbba833d6273.png">